### PR TITLE
Update Syntax and spellings.

### PR DESCRIPTION
**Pull request by DarkMatter188**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@DarkMatter188 Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-ws&pullRequestId=1161)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-ws&pullRequestId=1161) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

File: README.adoc [hunk 1]
```diff
@@ -51,10 +51,10 @@ The pipeline will build and release the "release" branch. It will also build a n
 Since the pipeline uses Docker, it's easy to:
 
 * Debug what went wrong on your local machine.
-* Test out a a tweak to your `test.sh` script before sending it out.
+* Test out for a tweak to your `test.sh` script before sending it out.
 * Experiment against a new image before submitting your pull request.
```
Comment (gregturn) [inline README.adoc:55]: I'd simply drop the extra "a".

File: README.adoc [hunk 2]
```diff
@@ -51,10 +51,10 @@ The pipeline will build and release the "release" branch. It will also build a n
...
 
-All of these use cases are great reasons to essentially run what Jenkins does on your local machine.
+All of these use cases  that are great reasons to know which essentially runs and  what Jenkins on your local machine.
```
Comment (gregturn) [inline README.adoc:58]: "All of these use cases are great reasons to run what Jenkins does on your local machine."

**Review by gregturn — CHANGES_REQUESTED** [review]

**Comment by gregturn** [comment]
Thanks @DarkMatter188. I've taken your patch, polished it, and merged to to `4.0.x`.

**Comment by gregturn** [comment]
Backported to `main`

**Pull request closed without merging**